docs/docs/integrations/chat/nvidia_ai_endpoints.ipynb

```diff
@@ -593,38 +493,24 @@
     "    top_p=1.0,\n",
     ")\n",
     "\n",
-    "#  Define a RunnableConfig object, with a `configurable` key. session_id determines thread\n",
-    "config = {\"configurable\": {\"session_id\": \"1\"}}\n",
-    "\n",
-    "conversation = RunnableWithMessageHistory(\n",
-    "    chat,\n",
-    "    get_session_history,\n",
-    ")\n",
-    "\n",
-    "conversation.invoke(\n",
-    "    \"Hi I'm Srijan Dubey.\",  # input or query\n",
-    "    config=config,\n",
-    ")"
+    "conversation = ConversationChain(llm=chat, memory=ConversationBufferMemory())"
```

Comment:
`ConversationChain` is deprecated in favor of `RunnableWithMessageHistory`. See [API reference](https://api.python.langchain.com/en/latest/chains/langchain.chains.conversation.base.ConversationChain.html). Why make this change?

Reply:
updated!